src/ApplePayGateway.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Omnipay\Vindicia;
+
+
+class ApplePayGateway extends AbstractVindiciaGateway
+{
+
+    /**
+     * Get the gateway parameters.
+     *
+     * @return array
+     */
+    public function getDefaultParameters()
```

Comment:
this is the same as the base class, so probably not necessary?

Reply:
or maybe the cert parameters should be added here

Reply:
I'm going to get rid of it because I don't believe it's necessary. Also, adding the certs here throws an error because getPemCertPath(), etc. would need to be in the capture and completeAuthorize methods. We don't need them for that.